src/lib/mutate-processor.ts

```diff
@@ -87,9 +87,19 @@ export async function mutateProcessor(
         Log.warn(actionMetadata, `Action failed: ${e}`);
         updateStatus("warning");
 
-        // Annoying ts false positive
+        let errorMessage = "";
+        try {
+          if (e.message && e.message.length > 0) {
```

Comment:
```suggestion
          if (e.message && e.message !== "[object Object]") {
```

Reply:
This update will catch any other [object Object] instances that come through. I've also updated the KFC Logs function so that for the specific case we were seeing, we'll get a more specific message of `Failed to get logs in KFC Logs function` instead of the general `An error occurred in the mutate action.`